src/main/java/me/fallenbreath/yamlang/YamlangExtension.java

```diff
@@ -50,4 +50,14 @@ public interface YamlangExtension
 	 * Default value: "UTF-8"
 	 */
 	Property<String> getCharset();
+
+	/**
+	 * Enables the rich translation support in yaml files
+	 * <p>
+	 * This option preserves lists added to the language file
+	 * <p>
+	 * Default value: false
+	 * @see <a href="https://docs.wispforest.io/owo/rich-translations/">OWO-LIB Rich Translations</a>
+	 */
+	Property<Boolean> getRichTranslations();
```

Comment:
The optional name "rich translations" is confusing. I don't think it's specific to OWO-LIB. I think a name like `owolibRichTranslations` that explictly show that it's a feature from OWO-LIB is better

Same for the `ConverterConfiguration`

Reply:
Also same change requested on the comment of this method, like

```diff
+Enables<a href="https://docs.wispforest.io/owo/rich-translations/">OWO-LIB Rich Translations</a> in the yaml translation files
-Enables the rich translation support in yaml files
```

Reply:
tldr don't use vague names like `richTranslations` or `allowLists`, use names like `owolibRichTranslations` that explictly shows it's the rich translation feature from owolib